src/sources/socket/tcp.rs

```diff
@@ -178,19 +201,44 @@ impl TcpSource for RawTcpSource {
 
         for event in events {
             if let Event::Log(ref mut log) = event {
-                log.try_insert(log_schema().source_type_key(), Bytes::from("socket"));
-                log.try_insert(log_schema().timestamp_key(), now);
-
-                let host_key = self
-                    .config
-                    .host_key
-                    .as_deref()
-                    .unwrap_or_else(|| log_schema().host_key());
-
-                log.try_insert(host_key, host.ip().to_string());
-                if let Some(port_key) = &self.config.port_key {
-                    log.try_insert(port_key.as_str(), host.port());
-                }
+                self.log_namespace.insert_vector_metadata(
```

Comment:
A `insert_standard_vector_source_metadata` function was recently added that will simplify the 2 vector metadata inserts a bit.
https://github.com/vectordotdev/vector/pull/15101